media/js/src/GridAsset.jsx

```diff
@@ -69,8 +82,8 @@ export default class GridAsset extends React.Component {
         const assetUrl = getAssetUrl(this.props.asset.id);
 
         return (
-            <div className="col-sm-4">
-                <h5 className="text-nowrap text-truncate">
+            <div className="col-sm-4 border-right border-white mb-2">
```

Comment:
This is looking better spacing-wise, but we can probably bump it up just a shade more. We can talk tomorrow.